WordPress/Classes/ViewRelated/Blog/Blog Dashboard/ViewModel/BlogDashboardViewModel.swift

```diff
@@ -93,6 +93,15 @@ class BlogDashboardViewModel {
         loadCardsFromCache()
     }
 
+    /// Update to display the selected blog.
+    func update(blog: Blog) {
+        BlogDashboardAnalytics.shared.reset()
+        self.blog = blog
+        self.blazeViewModel = DashboardBlazeCardCellViewModel(blog: blog)
+        self.loadCardsFromCache()
```

Comment:
`DashboardCardModel` has `dotComID` making sure that the Blaze card gets re-configured when blog changes.